### PR TITLE
Docs: Update 8.0 time series content

### PR DESCRIPTION
**Pull request by oddlittlebird**
- Added Time series panel content and links to What's new in 8.0.
- Updated content and added links.

**Note:** This is not a full rewrite of the content, there will be more PRs coming.

### CODE REVIEW

File: docs/sources/shared/visualizations/change-axis-link.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Stack series link
+---
+
+## Axis
+
```
Comment (achatterjee-grafana) [inline docs/sources/shared/visualizations/change-axis-link.md:6]: Needs a stem sentence

File: docs/sources/shared/visualizations/stack-series-link.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Stack series link
+---
+
```
Comment (achatterjee-grafana) [inline docs/sources/shared/visualizations/stack-series-link.md:4]: Needs a stem sentence.

File: docs/sources/whatsnew/whats-new-in-v8-0.md [hunk 3]
```diff
@@ -30,12 +30,24 @@ Shows discrete status or state transitions of something over time. For example d
 
 New visualization that allows categorical data display. Following the new panel architecture supports field config and overrides, common tooltip, and legend options.
 
+### Time series panel updates
+
+- The Time series is out of beta!  We are removing the `Beta`tag and graduating the Time series panel to a stable state.
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v8-0.md:35]:
```suggestion
The Time series is out of beta!  We are removing the `Beta`tag and graduating the Time series panel to a stable state.
```

File: docs/sources/whatsnew/whats-new-in-v8-0.md [hunk 4]
```diff
@@ -30,12 +30,24 @@ Shows discrete status or state transitions of something over time. For example d
...
+
 ### Panel editor updates
 
 - All options are now shown in a single pane.
 - You can now search panel options.
 - Value mapping has been completely redesigned.
 
+### Look and feel update
+
+Grafana 8 comes with a refreshed look and feel, including themes changed to be more accessible. Improved Grafana UI brings a number of adjustments and tweaks that will make the app even more fun to use. Under the hood, the new theme architecture enables us to bring more sophisticated themes control in the future.
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v8-0.md:49]:
```suggestion
Grafana 8 comes with a refreshed look and feel, including themes changed to be more accessible. The improved Grafana UI brings a number of adjustments and tweaks that make the application even more fun to use. Under the hood, the new theme architecture enables us to bring more sophisticated themes control in the future.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added a couple of comments. Overall, looks good.

File: docs/sources/shared/visualizations/stack-series-link.md (hunk 2 above)
Comment (oddlittlebird) [reply docs/sources/shared/visualizations/stack-series-link.md:4]: What do you mean?

File: docs/sources/shared/visualizations/change-axis-link.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/shared/visualizations/change-axis-link.md:6]: What do you mean by a stem sentence?

File: docs/sources/shared/visualizations/stack-series-link.md (hunk 2 above)
Comment (achatterjee-grafana) [reply docs/sources/shared/visualizations/stack-series-link.md:4]: Introduction, introducing ...

Comment (oddlittlebird) [reply docs/sources/shared/visualizations/stack-series-link.md:4]: In the title?

Or are you talking about before or after the heading? All the content that would be under this heading is in another topic. The heading exists to maintain consistency with the rest of the option entries and to hold the paragraph with the link pointing to the topic where the content lives.

Comment (achatterjee-grafana) [reply docs/sources/shared/visualizations/stack-series-link.md:4]: After the title.

Comment (oddlittlebird) [reply docs/sources/shared/visualizations/stack-series-link.md:4]: That is not needed for this content snippet. The snippet starts with the heading.

Comment (achatterjee-grafana) [reply docs/sources/shared/visualizations/stack-series-link.md:4]: Ah - ok. Got it.

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**